### PR TITLE
feat: [3.x] add account and contact models

### PR DESCRIPTION
**Pull request by djaiss**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,7031 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (asbiin) [inline yarn.lock:1]: We don't need yarn anymore, we can use npm directly, and the file `package-lock.json` is here to ensure package versions.

**Review by asbiin — CHANGES_REQUESTED** [review]
See my comment

File: yarn.lock (hunk 1 above)
Comment (djaiss) [reply yarn.lock:1]: We don’t use Yarn anymore?

Comment (asbiin) [reply yarn.lock:1]: With the 3.X branch, we could swith to npm yes

**Pull request merged**